HACKING.md

```diff
@@ -439,16 +430,43 @@ cockpit-bridge, etc.)
 
 This is how you would run cockpit-ws under gdb:
 
-    $ export G_DEBUG=fatal-criticals
-    $ export G_MESSAGES_DEBUG=cockpit-ws,cockpit-wrapper,cockpit-bridge
-    $ gdb --args ./cockpit-ws --port 10000 --no-tls
+    export G_DEBUG=fatal-criticals
+    export G_MESSAGES_DEBUG=cockpit-ws,cockpit-wrapper,cockpit-bridge
+    gdb --args ./cockpit-ws --port 10000 --no-tls
 
 And you can run cockpit-ws and cockpit-bridge under valgrind like this:
 
-    $ export G_DEBUG=fatal-criticals
-    $ export G_MESSAGES_DEBUG=cockpit-ws,cockpit-wrapper,cockpit-bridge
-    $ valgrind --trace-children=yes --trace-children-skip='*unix_chkpwd*' \
+    export G_DEBUG=fatal-criticals
+    export G_MESSAGES_DEBUG=cockpit-ws,cockpit-wrapper,cockpit-bridge
+    valgrind --trace-children=yes --trace-children-skip='*unix_chkpwd*' \
           ./cockpit-ws --port 10000 --no-tls
 
 Note that cockpit-session and cockpit-bridge will run from the installed
 prefix, rather than your build tree.
+
+## Manually installing the development dependencies
+
+If at all possible, use the cockpit/tasks container with toolbox/distrobox as
+documented above. Installing all necessary development packages manually on
+your machine is intrusive, error prone, difficult, and hard to debug.
```

Comment:
We have alert classes to make something look like a note, warning, error, on the website; it'd be great here.

```
{:.warning}
```

However, as this is rendered in both the website and directly on GitHub (and probably possibly read as a text file locally), I guess... so I guess we would need to use normal Markdown, which would mean we might want to italicize this.

So, at least emphasizing with italic would be this:

```suggestion
_If at all possible, use the cockpit/tasks container with toolbox/distrobox as
documented above. Installing all necessary development packages manually on
your machine is intrusive, error prone, difficult, and hard to debug._
```

Otherwise, prioritizing the website would look like this:

```suggestion
{:.warning}
If at all possible, use the cockpit/tasks container with toolbox/distrobox as
documented above. Installing all necessary development packages manually on
your machine is intrusive, error prone, difficult, and hard to debug.
```

Reply:
Thanks! I committed the italics version.